pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -107,6 +110,16 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 	_ = kubectl.AppendServiceHandler(func(svc *model.Service, ev model.Event) { m.updateHandler(svc) })
 	_ = kubectl.AppendInstanceHandler(func(si *model.ServiceInstance, ev model.Event) { m.updateHandler(si.Service) })
 	go kubectl.Run(stopCh)
+	opts := Options{
```

Comment:
is this function only called when the member cluster doesn't have an istiod itself?

Reply:
I think it will regardless, so we can use the feature flag to guard this for now.

Reply:
guard by `values.global.centralIstiod=false` and default is off

Reply:
Thanks!